### DISCUSSION TITLE
#3582 Forge 下载问题

### DISCUSSION THREAD

**Discussion by wuliaodexiaoluo** [Meloong-Git/PCL#3597] [讨论]
#3582

**Comment by shixiaogaoya** [comment]
我刚试了，hmcl也不行了

**Reply by wuliaodexiaoluo** [reply]
我不知道现在是不是节点问题
我刚被 BCMLAPI 拉入黑名单了。。。，现在不是浏览器 UA 都请求不了了

**Comment by wuliaodexiaoluo** [comment]
研究了半天也没研究出来啥问题，只能定为大量请求被 BMCLAPI 拉入黑名单

**Reply by 0916NINELEAVE** [reply]
抱歉这么久没看消息  我初三党平时碰不到电脑   我刚刚又试了一遍PCL下载同一个整合包不知道啥毛病又能下了（
那大佬你是不是……以后用不了BCMLAPI了     坏了只有大佬受伤的世界完成了

**Reply by wuliaodexiaoluo** [reply]
倒也不是，只是 Python Requests 的 UA 被 ban 了
PCL 还是没问题的，就算被 ban 掉了我网络直连 Mojang API 还是没问题的，而且这只是找 93 哥反馈一下就能解决的事情
